src/runtime/formal/ops/ops.cc

```diff
@@ -496,57 +492,29 @@ CVM_REGISTER_GLOBAL("cvm.runtime.formal.transpose")
         }
     }
 
-//<<<<<<< HEAD
-//    TShape xShape(x->shape, x->shape + x->ndim),
-//        yShape(y->shape, y->shape + y->ndim);
-//    for (uint32_t i = 0; i < getSize(x); i++) {
-//      auto xIndex = VectorIndex(xShape, i);
-//      TShape yIndex(y->ndim);
-//      for (int j = yIndex.ndim() - 1; j >= 0; j--) {
-//        yIndex[j] = xIndex[axes[j]];
-//      }
-//      y_data[ScalarIndex(yShape, yIndex)] = x_data[i];
-//=======
     for (Indices Y_indices(Y_shape); !Y_indices.End(); ++Y_indices) {
       Indices X_indices(X_shape);
       for (uint32_t i = 0; i < Y_indices.ndim(); ++i) {
         X_indices[axes[i]] = Y_indices[i];
       }
       Y_data[Y_indices.Index()] = X_data[X_indices.Index()];
-//>>>>>>> wlt
     }
-
-    // for(uint64_t i = 0; i < Y_shape.Size(); i++) {
-      // uint64_t o_i = i, in_i = 0;
-      // for(int j = ndim - 1; j >= 0; j--){
-        // uint64_t col = o_i % Y_shape[j];
-        // o_i /= Y_shape[j];
-        // int xi = 1;
-        // for(int tx = ndim-1; tx > axes[j]; tx--){
-          // xi *= X_shape[tx];
-        // }
-        // in_i += col * xi;
-      // }
-      // Y_data[i] = X_data[in_i];
-    // }
 });
 
 CVM_REGISTER_GLOBAL("cvm.runtime.formal.strided_slice")
 .set_body([](CVMArgs args, CVMRetValue *ret){
-    DLTensor *x = args[0];
-    DLTensor *y = args[1];
-    void *_attr = args[2];
-    auto *attr = static_cast<cvm::NodeAttrs*>(_attr);
-    auto &param = cvm::get<cvm::top::StridedSliceParam>(attr->parsed);
-
-    int32_t *x_data = static_cast<int32_t*>(x->data);
-    int32_t *y_data = static_cast<int32_t*>(y->data);
+    //DLTensor *x = args[0];
+    //DLTensor *y = args[1];
+    int32_t* x_data = CVMArg2Data<int32_t>(args[0]);
+    int32_t* y_data = CVMArg2Data<int32_t>(args[1]);
+    auto& param = CVMArg2Attr<cvm::top::StridedSliceParam>(args[2]);
     TShape begin = param.begin;
     TShape end = param.end;
     TShape stride = param.stride;
-    int ndim = y->ndim;
-    int32_t num_axis = x->ndim;
-    int64_t *dshp = x->shape;
+    int num_axis = CVMArgNdim(args[0]);
+    int ndim = CVMArgNdim(args[1]);
```

Comment:
It's no sense to define this function since `num_axis` equals the ndim of `xShape`. 
Same as the `ndim` variable.